tokenizers/src/utils/from_pretrained.rs

```diff
@@ -59,7 +59,7 @@ pub fn from_pretrained<S: AsRef<str>>(
         .into());
     }
 
-    let mut builder = ApiBuilder::new();
+    let mut builder = ApiBuilder::from_env();
```

Comment:
```suggestion
    let mut builder = ApiBuilder::from_env("HF_HOME");
```
should we not set that?

Reply:
Oh shit okay, I was looking at https://docs.rs/env_logger/latest/env_logger/struct.Builder.html#method.from_env, thanks for the heads up